<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Modbus-сканер:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="RTU_checkBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>RTU</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="TCP_checkBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TCP</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget111" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Устройство
                                                                                                </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="device_comboBox"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Стоп бит
                                                                                                </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="stop_bit_comboBox">
                   <item>
                    <property name="text">
                     <string>1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>2</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Скорость
                                                                                                    передачи в Бод
                                                                                                </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="speed_comboBox">
                   <item>
                    <property name="text">
                     <string>2400</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>4800</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>9600</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>14400</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>19200</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>38400</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>57600</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Паритет</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="parity_comboBox">
                   <item>
                    <property name="text">
                     <string>odd</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>even
                                                                                                    </string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>none
                                                                                                    </string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget222" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>IP-адрес</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ip_modbus_lineEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Порт</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="port_lineEdit"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Адрес ведомого устройства</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="address_device_comboBox">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Старт_регистр</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Число значений</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="number_comboBox">
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QPushButton" name="start_pushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Старт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exit_pushButton">
            <property name="text">
             <string>Закрыть</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="status_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>TCP_checkBox</sender>
   <signal>clicked()</signal>
   <receiver>RTU_checkBox</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RTU_checkBox</sender>
   <signal>clicked()</signal>
   <receiver>TCP_checkBox</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
